### PR TITLE
[Maintenance] Update github PR template

### PR DESCRIPTION
**Pull request by oallain**
| Q               | A
| --------------- | -----
| Branch?         | master 
| Bug fix?        | no
| New feature?    | no
| BC breaks?      | no
| Deprecations?   | no 
| Related tickets | 
| License         | MIT

Sylius 1.6 is "End of maintenance" since Apr 29, 2020, so we can update this template.

WDYT ?

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -9,7 +9,7 @@
 | License         | MIT
 
 <!--
- - Bug fixes must be submitted against the 1.6 or 1.7 branches (the lowest possible)
+ - Bug fixes must be submitted against the 1.7 branche (the lowest possible)
```
Comment (pamil) [inline .github/PULL_REQUEST_TEMPLATE.md:12]:
```suggestion
 - Bug fixes must be submitted against the 1.7 branch (the lowest possible)
```

Comment (oallain) [reply .github/PULL_REQUEST_TEMPLATE.md:12]: Sorry, "branche" is in French :wink:

**Comment by oallain** [comment]
Hi @pamil 

Travis is green, but in PR he is "pending". 
Have you already see this issue (and fix) https://github.com/travis-ci/travis-ci/issues/10204 ?

**Comment by pamil** [comment]
Thanks, @oallain! :1st_place_medal:

**Pull request merged**